spec/models/request_spec.rb

```diff
@@ -477,4 +477,85 @@
       expect(request.last_processed_by_user_id).to eq(instructor.id)
     end
   end
+
+  describe '#generate_email_response' do
+    let(:course_settings) do
+      CourseSettings.create!(
```

Comment:
This is a good test example, but probably belongs in a factory. 

The `email_template` should probably be an attribute of the class, or possibly the render email class.